style/radiolistview.less

```diff
@@ -49,6 +49,10 @@
 
 			&[ nav-selected=true ] {
 				background-color: #eaf3ff;
+
+				&[ darker-selected-background=true ] {
```

Comment:
Custom attributes for styling are highly unusual. I know `nav-selected` is an exception, maybe not even a valid one but I suggest we don't continue in this direction. The _regular_ way of doing this is with a css class.

Reply:
Yea actually I'm curious to understand what's a better way to do this, I struggled a bit with this approach and wasn't completely convinced at the end. 

This is of course to still be able to reuse `RadioListView` in `UsageReportConsent`. Another alternative I didn't explore much is replicating a radio list directly in `UsageReportConsent` because the options there happen to be binary and simple. 

> The regular way of doing this is with a css class.

Could you unpack this?

Reply:
Before we talk about ways of doing this... why would this listview have a different selected row color than the other listviews in the app? I recommend not doing that and checking with Sudhanshu now or letting it go through design review.

Reply:
Yea that too! Will mention it to Sudhanshu and update accordingly